Base/usr/share/man/man1/comm.md

```diff
@@ -0,0 +1,49 @@
+## Name
+
+comm - compare two sorted files line by line
+
+## Synopsis
+
+```**sh
+$ comm [options...] <file1> <file2>
+```
+
+## Description
+
+`comm` compares two **sorted** files specified by `file1` and `file2` line by line alphabetically. One of file1 and file2, but not both, can be `-`, in which case `comm` will read from `STDIN` for that file.
```

Comment:
Please refer to stdin as "standard input" in the manpages, similarly, "standard output" for stdout.